jwcrypto/tests.py

```diff
@@ -437,6 +437,30 @@ def test_jwkset(self):
             num += 1
         self.assertEqual(num, len(PrivateKeys['keys']))
 
+    def test_jwkset_get_keys(self):
+        # Test key set with mutiple keys
+        ksm = jwk.JWKSet.from_json(json_encode(PrivateKeys))
+        k1 = jwk.JWK.from_json(json_encode(PrivateKeys['keys'][0]))
+        kwargs = RSAPrivateKey.copy()
+        kwargs['kid'] = '1'
+        k2 = jwk.JWK(**kwargs)
+        num = 0
+        for item in ksm:
+            self.assertTrue(isinstance(item, jwk.JWK))
+            self.assertTrue(item in ksm)
```

Comment:
a bit redundant, you are iterating over ksm in which case this can fail?

Reply:
Good point. Removed.